cob_navigation_global/launch/amcl_node.xml

```diff
@@ -2,14 +2,16 @@
 <launch>
 
   <!-- combine front and rear scanner -->
-  <!--node pkg="topic_tools" type="relay" name="scan_front_relay" args="/base_laser_front/scan scan_combined"/>
-    <node pkg="topic_tools" type="relay" name="scan_rear_relay" args="/base_laser_rear/scan scan_combined"/-->
+  <node pkg="topic_tools" type="relay" name="scan_front_relay" args="/base_laser_front/scan scan_combined"/>
```

Comment:
I think we can get rid of this.

The use-case so far has been to feed amcl with all laser scans, but they have been processed individually. I don't see a reason for not using `scan_unified`...